testsuite/openshift/client.py

```diff
@@ -213,6 +213,18 @@ def select_operator(apiobject):
 
         return self.select_resource("pods", narrow_function=select_operator)
 
+    def get_apicast_operator(self):
+        """
+        Gets the selector for the apicast operator
```

Comment:
this says "selector", return description says "pod"... not sure what it returns :-)

Reply:
fixed